src/Sulu/Bundle/AdminBundle/Navigation/NavigationItem.php

```diff
@@ -300,6 +322,15 @@ public function copyChildless()
     }
 
     /**
+     * Create a new instance of current navigation item class
+     * @return NavigationItem
+     */
+    protected function newInstance()
+    {
+        return new NavigationItem($this->getName());
```

Comment:
The name of this method really confuses me... It's not really a new instance, neither it is a copy of the object...